mapserver/osm.map.mako

```diff
@@ -111,6 +107,127 @@ LAYER
 END
 % endfor
 
+# "column": "tourism",
+# "value": "zoo",
+# "type": "zoo",
+# "name": u"Zoos"
+LAYER
+    NAME "tourism_activity"
+    GROUP "zoo" #compatibily
+    EXTENT 473743 74095 839000 306400
+    TYPE POINT
+    STATUS ON
+    TEMPLATE fooOnlyForWMSGetFeatureInfo # For GetFeatureInfo
+    CONNECTIONTYPE postgis
+    PROCESSING "CLOSE_CONNECTION=DEFER" # For performance
+    CONNECTION "user=${dbuser} password=${dbpassword} host=${dbhost} dbname=osm"
+    DATA "geom FROM (SELECT regexp_replace(format(\'%s\', name), \'^$\', osm_id::text) AS display_name,name,osm_id,access,aerialway,amenity,barrier,bicycle,brand,building,covered,denomination,ele,foot,highway,layer,leisure,man_made,motorcar,\"natural\", operator, population, power, place, railway, ref, religion, shop, sport, surface, tourism, waterway, wood, way AS geom FROM planet_osm_point) AS foo USING UNIQUE osm_id USING srid=21781"
+    PROCESSING "NATIVE_FILTER=tourism in ('zoo','information','museum','viewpoint')"
+    LABELITEM "name"
+    PROJECTION
+        "init=epsg:21781"
+    END
+    TOLERANCE 10
+    TOLERANCEUNITS pixels
+    CLUSTER
+       MAXDISTANCE 40  # in pixels
```

Comment:
yes, the +xy labels is not so clear but I didn't know how to improve this: bigger white buffer around number? Use of another icon (it was my first attempt but it wasn't clear for my workmet)like the one below:

![regroup_gray](https://cloud.githubusercontent.com/assets/660191/15466594/aa7e13fe-20da-11e6-8e23-cdcea4f265d6.png)

Any idea?